doc/conf.py

```diff
@@ -0,0 +1,82 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+import os
+import sys
+import inspect
+
+__location__ = os.path.join(os.getcwd(), os.path.dirname(
+    inspect.getfile(inspect.currentframe())))
```

Comment:
Why not use `__file__`?

Reply:
you mean as a name? sure, works. changed it.

Reply:
Nooo 😅 The special variable `__file__` is automatically set when loading a module/script from a file: https://docs.python.org/3/reference/import.html#file__

So you should get the absolute path to the directory containing the file like this:
```python
__location__ = os.path.dirname(os.path.abspath(__file__))
```

Reply:
haha i kinda thought so